helpers/calendar/ratios_and_utilization.py

```diff
@@ -30,7 +30,7 @@ def get_analytics_ratios(self, query, start, end):
         app_events_list = []
 
         for room in rooms:
-            checkins, cancellations, calendar_events_list, app_events_list = RoomAnalyticsRatios().retrieve_cancellations_and_checkins_for_room(room['room_id'], # noqa 
+            checkins, cancellations, calendar_events_list, app_events_list = RoomAnalyticsRatios().retrieve_cancellations_and_checkins_for_room(room['room_id'],  # noqa
```

Comment:
Is it possible to get rid of `#noqa`? Something like:
```
(
                checkins, cancellations, calendar_events_list, app_events_list
            ) = RoomAnalyticsRatios(
            ).retrieve_cancellations_and_checkins_for_room(
                room['room_id'],
                start_date,
                day_after_end_date,
                checkins,
                cancellations,
                calendar_events_list,
                app_events_list
            )
```

Reply:
I'm getting an invalid syntax error.
<img width="778" alt="Screenshot 2019-04-15 at 10 05 07 AM" src="https://user-images.githubusercontent.com/13919080/56120426-f8ae0300-5f65-11e9-8d3a-d4efc7ef29d2.png">

Reply:
Remove the enclosing braces in line 33

Reply:
something like this. But I think this is not so readable.
```suggestion
            (
                checkins, cancellations, calendar_events_list, app_events_list
            ) = RoomAnalyticsRatios(
            ).retrieve_cancellations_and_checkins_for_room(
                room['room_id'],
                start_date,
                day_after_end_date,
                checkins,
                cancellations,
                calendar_events_list,
                app_events_list
            )
```

Reply:
okay no problem. You can leave it just like that. Its readable that way anyway!